### PR TITLE
add batchelor extension for R-bundle-Bioconductor 3.9

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.9-foss-2019a-R-3.6.0.eb [hunk 1]
```diff
@@ -101,10 +101,8 @@ exts_list = [
         'checksums': ['d72cd4a1ba3fb14629aad3685202388384673ef06d2d22451c4c3d174a85e8e8'],
     }),
     ('preprocessCore', '1.46.0', {
-        'checksums': ['a062860681595930b98c98403d44242284a0692da197b7d42eac6e0a05b13e46'],
-        # disable internal threading to avoid crash in pthread_create
-        # see also https://github.com/bmbolstad/preprocessCore/issues/1
```
Comment (boegel) [inline easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.9-foss-2019a-R-3.6.0.eb]: @smoors Please restore this comment (that's bug in `--inject-checksums` we have to fix, see also https://github.com/easybuilders/easybuild-framework/issues/3082)

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node375.hydra.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/ec1e1d363f9b87b30502e1d877b95965 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node3124.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/1257d6cc853fe38468a872592f027efc for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/ef32fe07259229385d0b2f35cfe95501 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3124.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/d801d9a979fc9e944053ee13fd40dbda for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2568.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/8370a75d1a19a48b3091ce152eb5a547 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @smoors!

**Pull request merged**